src/utilities/permissions.ts

```diff
@@ -760,6 +764,9 @@ const queryPermissions: Record<GQLKeys, (user: User | null, ...args: any[]) => b
   INSERT_EXPANSION_SEQUENCE_TO_ACTIVITY: (user: User | null): boolean => {
     return isUserAdmin(user) || getPermission([Queries.INSERT_SEQUENCE_TO_SIMULATED_ACTIVITY], user);
   },
+  MIGRATE_PLAN_TO_MODEL: (user: User | null): boolean => {
```

Comment:
@duranb review permissions here, thinking we need to add in a plan-specific check.

Reply:
Looks like both `check_model_compatability` and `migrate_plan_to_model` have fine grained permissions. Any idea which `getRole*` function to use here? Both of these are hasura functions.

Reply:
My two cents is that I think `getRolePlanPermission` might be the most appropriate here since it's applying changes to the plan.  `getRoleModelPermission` was intended for getting permission for updating a model's metadata. Whoever created those functions might have a more definitive answer though.

Reply:
I went ahead and updated the permission. I haven't been able to test it so if @dandelany or @mattdailis could run a couple of tests, that would be appreciated.